features/bevragen/persoon/gezag/gba/gezag-meerderjarige-gba.feature

```diff
@@ -0,0 +1,192 @@
+#language: nl
+
+@gba
+Functionaliteit: gezagsrelaties van een meerderjarige
+
+
+  Rule: een meerderjarige krijgt voor een minderjarig kind met twee ouders met gezag de gezagsrelatie naar beide ouders geleverd
+
+    Scenario: beide ouders hebben gezag over het minderjarige kind van bevraagde persoon
+      Gegeven voor de persoon met burgerservicenummer '000000024' gelden de volgende gezagsrelaties
+      | bsnMinderjarige | soortGezag | bsnMeerderjarige |
+      | 000000012       | OG2        | 000000024        |
+      En voor de persoon met burgerservicenummer '000000012' gelden de volgende gezagsrelaties
+      | bsnMinderjarige | soortGezag | bsnMeerderjarige |
+      | 000000012       | OG2        | 000000024        |
+      | 000000012       | OG2        | 000000048        |
+      Als gba personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 000000024                       |
+      | fields              | gezag                           |
+      Dan heeft de response een persoon met een 'gezag' met de volgende gegevens
+      | naam                             | waarde                    |
+      | type                             | TweehoofdigOuderlijkGezag |
+      | minderjarige.burgerservicenummer | 000000012                 |
+      En heeft 'gezag' een 'ouder' met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000024 |
+      En heeft 'gezag' een 'ouder' met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 000000048 |
+
+
+  Rule: een meerderjarige krijgt voor een minderjarig kind met één ouder met gezag de gezagsrelatie naar deze ouder geleverd
```

Comment:
naar zichzelf geleverd? Het kan niet iemand anders zijn toch? Suggestie

een meerderjarige ouder krijgt voor een minderjarig kind met één ouder met gezag de gezagsrelatie naar zichzelf geleverd

Reply:
deze rule, ook de variant die @CathyDingemanse voorstelt is functioneel niet helemaal correct.
Er kunnen namelijk ook 2 ouders zijn, waarvan er één gezag heeft en gezag van de andere ouder kan bevraagd worden. Die krijgt dan geen gezag.
Ingewikkeld om dat correct op te schrijven... Zoiets?: "een meerderjarige die als enige gezag heeft over een minderjarig kind krijgt de gezagsrelatie naar zichzelf geleverd"